packages/vscode/README.md

```diff
@@ -0,0 +1,53 @@
+# Kubernetes Secrets for Zowe Explorer
+
+## Installing
+
+Use one of the following methods to install the plug-in:
+
+- Install the plug-in to VS Code from the marketplace
+
+  1. On the left most side of the VS Code window in your workspace, click on the extensions icon
+
+  1. On top of the extensions view, click on the search bar and search for `Kubernetes Secrets for Zowe Explorer`.
+
+  1. Click on the install button
+
+- Build the plug-in from source and install it into your Zowe CLI implementation.
+
+  Use the build from source method when you want to install the plug-in to VS Code using the most current binaries and modify the behavior of the plug-in. For example, you want to create a new command and use the plug-in with the command that you created.
+
+  For more information, see [Building from source](#building-from-source).
+
+## Building from source
+
+To build from source, follow these steps:
+
+1. Clone this repository
+1. Install, build, and package the project with the following command:
+
+   ```bash
+     npm install && npm build && npm package
+   ```
+
+1. View the `dist/` folder in the project, this should contain a file named `Zowe.secrets-for-kubernetes.vsix`
```

Comment:
I would tweak this sentence a little. Either:

- View the `dist/` folder in the project. This should contain ...
- View the `dist/` folder in the project, which should contain ...